skr-api/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM rust:1.71.0-slim-bullseye as build
+RUN apt-get update && apt-get install -y libssl-dev pkg-config protobuf-compiler curl gnupg2 build-essential clang
+RUN curl -L https://download.01.org/intel-sgx/sgx_repo/ubuntu/intel-sgx-deb.key | apt-key add -
+RUN echo 'deb [arch=amd64] https://download.01.org/intel-sgx/sgx_repo/ubuntu focal main' > /etc/apt/sources.list.d/intel-sgx.list
+RUN apt-get update && apt-get install -y libtdx-attest-dev libtss2-dev
+COPY Cargo.* /build/
+COPY build.rs /build/
+COPY protos /build/protos
+COPY src /build/src
+WORKDIR /build
+RUN cargo b --release --bin skr-api
+
+FROM debian:bullseye-slim as libraries
+RUN apt-get update && apt-get install -y libtss2-tctildr0 libtss2-esys-3.0.2-0
```

Comment:
Why do we need tss libs for `skr-api`?

Reply:
yes, this can be fixed by creating a workspace and compiling skr-api individually, fixed in [this commit](https://github.com/confidential-containers/cloud-api-adaptor/pull/1237/commits/d1f52b25318bf85f3288ea255dc98c42315c34ee)